### DISCUSSION TITLE
Question about Windows Store apps

### DISCUSSION THREAD

**Discussion by nitishvijai** [microsoft/winget-pkgs#331] [General] [answered]
Will Windows Store applications be supported with winget? I'd like to be able to submit a pull request for one of my Windows apps on the Store.

On a side note, props for all the hard work you all have done on this so far!

**Comment by atomic1fire** [comment]
I think it would be even better if every (purchased/free) windows store app was treated as part of the repository, personally. Especially factoring in "apps" like Ubuntu, which exists for WSL.

Although I don't know how purchases would work unless winget relies on the user's microsoft account.

But it does sound like this is something they're aware of.

https://github.com/microsoft/winget-cli/issues/117

**Comment by nitishvijai** [comment]
@atomic1fire Exactly! I think this in itself would help improve the winget repository immensely.

**Comment by jvintzel** [comment]
We are tracking this issue in the open source client repository.  https://github.com/microsoft/winget-cli/issues/117

**Comment by vedantmgoyal9** [comment]
Installing apps directly from Microsoft Store is now supported. However, there are still some issues the team is working upon.

**Comment by denelon** [comment, marked as answer]
Installing packages from the Microsoft Store is now enabled. The default configuration includes the "msstore" source to install free apps rated e for everyone.
